site/src/components/TableCellData/TableCellData.tsx

```diff
@@ -0,0 +1,35 @@
+import { makeStyles } from "@material-ui/core/styles"
+import React from "react"
+import { Stack } from "../Stack/Stack"
+
+export const TableCellData: React.FC = ({ children }) => {
+  return <Stack spacing={0}>{children}</Stack>
+}
+
+export const TableCellDataPrimary: React.FC<{ highlight?: boolean }> = ({
+  children,
+  highlight,
+}) => {
+  const styles = useStyles({ highlight })
+
+  return <span className={styles.primary}>{children}</span>
+}
+
+export const TableCellDataSecondary: React.FC = ({ children }) => {
+  const styles = useStyles()
+
+  return <span className={styles.secondary}>{children}</span>
+}
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
If you want, you can type this whole style hook so that you don't have to redeclare the type every time you use the `highlight` prop below:

````
interface StyleProps {
  highlight?: boolean
}

const useStyles = makeStyles<Theme, StyleProps>((theme) => ({
....
````